.pyup.yml

```diff
@@ -1,3 +1,4 @@
+update: insecure
```

Comment:
Do we also need `update: security` nested underneath the python-requirements.txt entry?

Reply:
No, that's only if there are multiple files and you want to specify different update methods per file. It's also `update: insecure`, which sounds backwards, but that is the config for only updating insecure packages.